.circleci/config.yml

```diff
@@ -64,11 +67,23 @@ workflows:
   version: 2
   e2e:
     jobs:
-      - e2e-minikube
+      - e2e-minikube:
+          matrix:
+            parameters:
+              minikube-version: # https://github.com/kubernetes/minikube/releases
+                - v1.22.0
+                - v1.21.0
+                - v1.20.0
```

Comment:
Changelog says we're testing with 3 the latest K8s versions, while the matrix is about minikube versions which is not the same.

I saw you debugged the latest `minikube` (and all the cri-o stuff) as it doesn't work anymore with the default engine as I understand.

To assist, the docker driver looks to be working with the latest minikube with no customizations: 
https://github.com/StackStorm/stackstorm-k8s/compare/test-minikube?expand=1#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47

We can use that to pin the latest minikube, and add 3 K8s versions to the matrix then as originally thought.

Reply:
Yep, thanks for the help on that, I'm iterating...

Reply:
I'm not sure why one of the checks isn't returning status, but, I think we're passing now.

Reply:
> We can use that to pin the latest minikube, and add 3 K8s versions to the matrix then as originally thought.

Ah ... I'll do this next.